thirdeye/thirdeye-frontend/app/pods/components/update-table/component.js

```diff
@@ -5,6 +5,8 @@ import modelsTable from 'ember-models-table/components/models-table';
  * the table's selected items should reflect this
  */
 export default modelsTable.extend({
+
+  classNames: ['events-table'],
```

Comment:
the `metrics-table` also uses `update-table`. Can you double check that there are no side effects here?

Reply:
sure thing.